.github/workflows/build-release-binaries.yml

```diff
@@ -0,0 +1,98 @@
+name: Release Bitshares-ui
+
+on:
+    push:
+        tags:
+            - "*.*.*"
+            # branches: [testaction]
+
+jobs:
+    build:
+        runs-on: macOS-latest
+
+        strategy:
+            matrix:
+                node-version: [16.13.x]
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v1
+              with:
+                  node-version: ${{ matrix.node-version }}
+            - run: echo "PackageVersion=$(cat package.json | sed 's/,/\n/g' | grep '\"version\":' | sed 's/:/\n/g' | sed '1d' | sed 's/ //g' | sed 's/\"//g' | sed 's/}//g')" >> $GITHUB_ENV
+            - name: Update package.json
+              run: |
+                  sed -i -e "s/${{ env.PackageVersion }}/${{ github.ref_name }}/g" package.json
+            - run: yarn
+            - name: Build package
+              run: npm run prepackage
+            - name: Build dmg
+              run: ./node_modules/.bin/electron-builder -p never --mac dmg
+            - name: Build deb
+              run: ./node_modules/.bin/electron-builder -p never --linux deb
+            - name: Build exe
+              run: ./node_modules/.bin/electron-builder -p never --windows
+            - name: Calc hash
+              id: calc_hash
+              run: |
+                  echo "::set-output name=dmghash::${{ hashFiles('build/binaries/**.dmg') }}"
+                  echo "::set-output name=debhash::${{ hashFiles('build/binaries/**.deb') }}"
+                  echo "::set-output name=exehash::${{ hashFiles('build/binaries/**.exe') }}"
+            - name: Create Body
+              id: create_body
+              uses: xiangxn/generate_release_notes@v1
```

Comment:
Can we have this action maintained in the bitshares organization?

Reply:
See https://github.com/bitshares/bitshares-ui/pull/3416/files#r771720347

Reply:
Of course, this is originally MIT

Reply:
> Of course

Please do it then. Thanks.
- Move or fork the action into https://github.com/bitshares
- use new URL here.